code/supervised/Logistic_Regression/logistic_ex1.py

```diff
@@ -0,0 +1,52 @@
+from sklearn.linear_model import LogisticRegression
+import numpy as np
+import random
+
+#defines the classification for the training data.
+def true_classifier(i):
+    if i >= 700:
+        return 1
+    return 0
+
+#Generate a random dataset which includes random scores from 0 to 1000.
+x = np.array([ random.randint(0,1000) for i in range(0,1000) ])
+
+#The model will expect a 2D array, so we must reshape
+#For the model, the 2D array must have rows equal to the number of samples,
+#and columns equal to the number of features.
+#For this example, we have 1000 samples and 1 feature.
+x = x.reshape((-1, 1))
+
+#For each point, y is a pass/fail for the grade. The simple threshold is arbitrary,
+#and can be changed as you would like. Classes are 1 for success and 0 for failure
+y = [ true_classifier(x[i][0]) for i in range(0,1000) ]
+
+
+#Again, numpy wants a 2d array, so we convert.
```

Comment:
DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True).

I got this warning also.

Reply:
I think this is also from different versions, but I might be wrong.  I'll look into what is the latest practice.

Reply:
Switched to a simple numpy array, please verify the warning goes away when you get the chance.